bots/pr/src/test/java/org/openjdk/skara/bots/pr/PullRequestBotFactoryTest.java

```diff
@@ -112,7 +112,8 @@ public void testCreate() {
                             "integrator1",
                             "integrator2"
                           ],
-                          "reviewCleanBackport": true
+                          "reviewCleanBackport": true,
+                          "reviewMergePullRequest": true,
```

Comment:
I think `reviewMerge` is enough for the configuration option. The word "PullRequest" seems redundant in the context of pr bot.